apps/labs/posts/categories.json

```diff
@@ -5,6 +5,7 @@
   "Community",
   "Developer workflows",
   "Funding",
+  "GPU",
```

Comment:
I'm assuming that the addition of this category was previously discussed, reviewed and approved

Reply:
Given that there was an additional `CuPy` category _also_ added, and then removed after subsequent discussion, I'm assuming that yes, the `GPU` category add is good to remain.

Reply:
That one looked like a good idea to me, yes. I didn't see anyone else comment on it.

Reply:
Are there any other posts in the archive that should have `GPU` backfilled onto?